spec/legacy/functional/my_controller_spec.rb

```diff
@@ -120,7 +120,7 @@
       end
 
       it { is_expected.to set_flash.to /reset/ }
-      it { is_expected.to redirect_to '/my/account' }
+      it { is_expected.to redirect_to '/my/access_token' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.